legacy/src/api/payment-requests/payment-requests.md

```diff
@@ -103,19 +103,30 @@ version (documented on this page) and the "legacy" version (documented at
 
 {% h4 Optional Fields %}
 
-|     Field      |  Type  |                                         Description                                          |
-| -------------- | ------ | -------------------------------------------------------------------------------------------- |
-| bitcoinAddress | String | ★  Address to send Bitcoin, when the "assetType" is `bitcoin.*`                              |
-| cennzAddress   | String | ★  Address to send CPay, when the "assetType" is `cennznet.*`                                |
-| wavesAddress   | String | ★  Waves address to send Zap tokens, when the "assetType" is `zap.*`                         |
-| productCodes   | Array  | Supported product codes for the payment request, when the "assetType" is `epay.nzd.*`        |
-| collectionIds  | Array  | Supported collectionIds for the payment request, when the "assetType" is `centrapay.token.*` |
+|     Field            |  Type  |                                         Description                                                                 |
+| -------------------- | ------ | ------------------------------------------------------------------------------------------------------------------- |
+| bitcoinAddress       | String | ★  Address to send Bitcoin, when the "assetType" is `bitcoin.*`                                                     |
+| cennzAddress         | String | ★  Address to send CPay, when the "assetType" is `cennznet.*`                                                       |
+| wavesAddress         | String | ★  Waves address to send Zap tokens, when the "assetType" is `zap.*`                                                |
+| productCodes         | Array  | Supported product codes for the payment request, when the "assetType" is `epay.nzd.*`                               |
+| acceptedCollections  | Array  | [Accepted Collections](###accepted-collection) for the payment request, when the "assetType" is `centrapay.token.*`|
```

Comment:
This link does not work when I run the site locally

Reply:
This has been fixed :)